src/scheduler.cpp

```diff
@@ -116,7 +116,7 @@ void CScheduler::scheduleFromNow(CScheduler::Function f, int64_t deltaMilliSecon
 
 void CScheduler::MockForward(boost::chrono::seconds delta_seconds)
 {
-    assert(delta_seconds.count() > 0 && delta_seconds.count() < 3600);
+    assert(delta_seconds.count() > 0 && delta_seconds < boost::chrono::hours{1});
```

Comment:
Maybe add a very brief comment *why* 1 hour?

Reply:
oops. forgot to address this, but going to leave it as is now.